### PR TITLE
Hao/fetch oppurturnity options

### PR DESCRIPTION
**Pull request by haophn01**
This PR is for the opportunity stats, 
- I move the file OpportunityStats from explore UI to message UI.
- The query is already done with value process as well.

### CODE REVIEW

File: client/src/ui/message/OpportunityStats.tsx
Comment (MekhiHart) [inline client/src/ui/message/OpportunityStats.tsx:1]: can you replace the mockData with the actual data? You don't need the mock data

File: client/src/app/(main)/message/company/[conversationId]/page.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { OpportunityStats } from "@/ui/message/OpportunityStats";
 import { Conversation } from "@/ui/message/Conversation";
 export default function Page({
   params,
 }: {
-  params: { conversationId: string };
+  params: { conversationId: string, opportunity_id: string };
 }) {
-  const { conversationId } = params;
+  const { conversationId, opportunity_id } = params;
```
Comment (MekhiHart) [inline client/src/app/(main)/message/company/[conversationId]/page.tsx:9]: The opportunity ID will not be passed in as  url param. So you can just remove this

File: client/src/app/(main)/message/company/[conversationId]/page.tsx [hunk 2]
```diff
@@ -15,7 +15,8 @@ export default function Page({
       }}
     >
       <Conversation conversationId={conversationId} />
-      <OpportunityStats />
+      {/* TODO: NEED ACTUAL A WAY TO PASS IN THE opportunity_id */}
+      <OpportunityStats opportunity_id={opportunity_id} />
```
Comment (MekhiHart) [inline client/src/app/(main)/message/company/[conversationId]/page.tsx:19]: you can just pass in a random id for this

**Review by MekhiHart — CHANGES_REQUESTED** [review]

File: client/src/ui/message/OpportunityStats.tsx
Comment (haophn01) [reply client/src/ui/message/OpportunityStats.tsx:1]: it is actual data

File: client/src/app/(main)/message/company/[conversationId]/page.tsx (hunk 1 above)
Comment (haophn01) [reply client/src/app/(main)/message/company/[conversationId]/page.tsx:9]: ok

File: client/src/app/(main)/message/company/[conversationId]/page.tsx (hunk 2 above)
Comment (haophn01) [reply client/src/app/(main)/message/company/[conversationId]/page.tsx:19]: ok

File: client/src/ui/message/OpportunityStats.tsx
Comment (haophn01) [reply client/src/ui/message/OpportunityStats.tsx:1]: It is just I did not change the variable name.

Comment (MekhiHart) [reply client/src/ui/message/OpportunityStats.tsx:1]: can you change the variable name then since it's not mock data anymore?

**Review by MekhiHart — APPROVED** [review]

File: client/src/ui/message/OpportunityStats.tsx
Comment (haophn01) [reply client/src/ui/message/OpportunityStats.tsx:1]: okie sure

**Pull request merged**